src/components/http/HttpSettings.tsx

```diff
@@ -211,7 +211,6 @@ export const HttpSettingsForm = ({ isEditor }: Props) => {
               name="settings.http.headers"
               disabled={!isEditor}
               label="header"
-              limit={10}
               validateName={validateHTTPHeaderName}
               validateValue={validateHTTPHeaderValue}
```

Comment:
There is also a proxy header limit of 10 below - should we remove that one too?

Reply:
Yeah good call